.github/workflows/test-external-providers.yml

```diff
@@ -0,0 +1,89 @@
+name: Test External Providers
+
+on:
+  push:
+    branches: [ main ]
+  pull_request:
+    branches: [ main ]
+
+jobs:
+  test-external-providers:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
```

Comment:
(No action required here) would be nice to pin these going forward: https://github.com/meta-llama/llama-stack/pull/1776